manifest.json

```diff
@@ -21,6 +21,11 @@
       "description": "Remove all BIDS info from files before curating",
       "type": "boolean",
       "default": false
+    },
+    "entire_project": {
+      "description": "Run bids curation on the entire project",
+      "type": "boolean",
+      "default": true
```

Comment:
I'm guessing this is set to true so that it won't confuse current users?

Reply:
Yeah, seemed slightly clearer to name the property this way and make it the default.